src/IoTHub.Portal.Shared/Models/v1.0/UserModel.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) CGI France. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace IoTHub.Portal.Shared.Models.v10
+{
+
+    public class UserModel
+    {
+        public string Id { get; set; }
+        public string Email { get; set; }
+        public string Name { get; set; }
+        public string Forename { get; set; }
+    }
```

Comment:
What is the difference with the UserDetails?

Reply:
The difference with 'UserDetailsModel' is that in this class we don't have the list of accessControls (commented for the moment as not yet implemented).

Reply:
Outdated (before the architecture change)